queue_services/business-bn/src/business_bn/resources/business_bn.py

```diff
@@ -81,26 +81,28 @@ def worker():
 
         current_app.logger.info(f"received ce: {ce!s}")
 
-        event_message = json.loads(msg.data.decode("utf-8"))
-        current_app.logger.debug("Event Message Received: %s", event_message)
-        process_event(event_message)
+        process_event(ce)
         return {}, HTTPStatus.OK
 
     except OperationalError as err:
-        current_app.logger.error("Queue Blocked - Database Issue: %s", json.dumps(event_message), exc_info=True)
+        current_app.logger.error("Queue Blocked - Database Issue: %s", json.dumps(ce), exc_info=True)
```

Comment:
here and below, `json.dumps(ce)` will throw an error. You can just pass it in as 'ce'